seqr/views/apis/users_api_tests.py

```diff
@@ -163,8 +163,9 @@ def _test_create_project_collaborator(self, response, create_url=None, mock_send
         new_collab = next(collab for collab in collaborators if collab['email'] == 'test@test.com')
         self.assertEqual(new_collab['username'], username)
         self.assertEqual(new_collab['displayName'], 'Test Name Update')
-        self.assert_json_logs(user, [('update User 20', {'dbUpdate': {
-            'dbEntity': 'User', 'entityId': 20, 'updateFields': ['last_name'], 'updateType': 'update'}})])
+        new_user_model = User.objects.get(username=new_collab['username'])
```

Comment:
the primary key offset is less deterministic for some reason with the addition of an additional database, so removing references in the tests to hardcoded auto-generated db IDs